en/Case_study/yourFirstModule.md

```diff
@@ -451,6 +451,6 @@ Once your module is released, you may think that your work is over. However, the
 
 When done with developing and testing your module, you can release it for developers by deploying to the Maven repository using instructions at: http://om.rs/newdevtagging
 
-For end users, you can upload it to the module repository http://modules.openmrs.org which is available to everyone. Read more about our rules and regulations for it here: http://om.rs/newdevmodrepo 
+For end users, you can upload it to the [Add-ons index](https://addons.openmrs.org) which is available to everyone. Read more about our rules and regulations for it here: http://om.rs/newdevmodrepo 
```

Comment:
Did you intentionally drop the words "module repository"?

Reply:
Yes I did @dkayiwa . Now, I think it's wrong as you cant actually upload to add-ons index. Should I change this?